### PR TITLE
add inputmask-multi v1.2.0

### LINKED ISSUE #7549: [Request] Add _inputmask-multi
**Library name: Inputmask Multi** 
**Git repository url: https://github.com/andr-04/inputmask-multi**
**npm package url(optional):** 
**License(s):**
**Official homepage: http://andr-04.github.io/inputmask-multi/**
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #7549 
@nolsherry Please help me review this pull request, thank you.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/andr-04/inputmask-multi
 * Official website (optional, not the repository): http://andr-04.github.io/inputmask-multi/
 * NPM package url (optional): https://www.npmjs.com/package/inputmask-multi
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 138
   - Count of watchers: 18
   - Count of forks: 50
 * NPM download stats (optional):
   - Downloads in the last day: 2
   - Downloads in the last week: 7
   - Downloads in the last month: 18

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

with npm autoupdate config, close #7549, cc @andr-04

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@pvnr0082t don't assign to @nolsherry until she added more than 10 libs.

**Comment by PeterDaveHello** [comment]
@x09326 please take this.

File: ajax/libs/inputmask-multi/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "ecosystem:jquery"
+  ],
+  "homepage": "https://github.com/andr-04/inputmask-multi",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/andr-04/inputmask-multi.git",
+    "basePath": "",
+    "files": [
+      "js/jquery.inputmask-multi*"
+    ]
```
Comment (x09326) [inline ajax/libs/inputmask-multi/package.json]: If this lib was also put on npm, I think you can use npm auto-update, that would be better.
And the ```homepage``` should be http://andr-04.github.io/inputmask-multi/

Comment (PeterDaveHello) [inline ajax/libs/inputmask-multi/package.json]: maybe you can open an issue to remind author publish new version on npm.

Comment (andr-04) [inline ajax/libs/inputmask-multi/package.json]: Sorry for my mistake. I've updated the package in npm just now.

Comment (pvnr0082t) [inline ajax/libs/inputmask-multi/package.json]: @andr-04 Thank you for your help.

**Comment by pvnr0082t** [comment]
@x09326 I use ```npm``` autoupdate config instead of ```git``` autoupdat and ```homepage``` filed is fixed. Please help me review this pull request again, thank you.

**Comment by x09326** [comment]
@PeterDaveHello I think this PR is okay.
Please help me review it, thank you.

**Comment by PeterDaveHello** [comment]
@x09326 please ping me after the CI test passed.

File: ajax/libs/inputmask-multi/package.json (hunk 1 above)
Comment (PeterDaveHello) [inline ajax/libs/inputmask-multi/package.json]: Thanks @andr-04

**Comment by x09326** [comment]
@PeterDaveHello The CI test has passed, please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
Thanks.

**Comment by PeterDaveHello** [comment]
@pvnr0082t please put the info about npm auto-update in the title and first line of commit message. It's not too long in this case.

**Pull request merged**